src/raftstore/store/snap.rs

```diff
@@ -966,6 +972,7 @@ impl Snapshot for Snap {
             meta_file.sync_all()?;
         }
         fs::rename(&self.meta_file.tmp_path, &self.meta_file.path)?;
+        fsync_by_path(&self.dir_path)?;
```

Comment:
I suggest to sync it at L966 first and then here. IO operation can be reordered.

Reply:
Done